caffe2/opt/converter.cc

```diff
@@ -295,15 +297,26 @@ repr::NNModule convertToNNModule(caffe2::NetDef &net, std::unordered_map<std::st
   auto bbNode =
       cfg.createNode(util::make_unique<repr::BasicBlockType<repr::NNGraph>>());
 
+  std::unordered_map<std::string, repr::NNGraph::NodeRef> entryNodeMap;
+  std::unordered_map<std::string, repr::NNGraph::NodeRef> exitNodeMap;
+
+  for (auto external_input : net.external_input()) {
+    entryNodeMap[external_input] = nullptr;
+  }
+  for (auto external_output : net.external_output()) {
+    exitNodeMap[external_output] = nullptr;
+  }
+
   for (auto &op : *net.mutable_op()) {
```

Comment:
not related to the diff, but normally mutable data is not passed by a reference but by a pointer (according to Google c++ code style)